english/challenge-marshmellow-to-rock-off/light-route/up-or-down/go-down.md

```diff
@@ -0,0 +1,2 @@
+You descend into madness. Your only light is the glowing of marshmallow lights. You are weightless. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.